sonar-plugin/src/main/java/io/ecocode/javascript/checks/AvoidHighAccuracyGeolocation.java

```diff
@@ -26,7 +26,7 @@
 @JavaScriptRule
 @TypeScriptRule
 @Rule(key = AvoidHighAccuracyGeolocation.RULE_KEY)
-@DeprecatedRuleKey(repositoryKey = "ecocode-javascript", ruleKey = "EC8")
+// @DeprecatedRuleKey(repositoryKey = "ecocode-javascript", ruleKey = "EC8")
```

Comment:
This deprecated rule breaks the SonarQube start.
I hadn't detected it in the PR that covered this topic: I've disabled it here to check that the proof of concept is working properly. Thanks to integration testing, such cases should no longer occur.

Reply:
for me, it's important that it can work with Deprecated annotation ... we have this use case 😰

Reply:
yes, as I said this is not normal and I use it in this draft PR to check if the health check works